contracts/OXOKoskar.sol

```diff
@@ -0,0 +1,137 @@
+pragma solidity 0.4.15;
+
+//Game board is a [0-8] array 
+//Game starts when the first player makes a valid turn.
+//Turn is made by calling makeTurn() with cell index [0-8] as an argument
+//Valid turn is: 
+//  1) marking an empty cell 
+//  2) bid is >= 1000wei + previous bid
+//  3) not a second in a row by this player
+//After a win or a standoff the contract is cleared and ready for a new game
+
+
+contract OXOKoskar {
+    
+    bytes1[9] public gameBoardCells;
+    address public whosTurn;
+    uint public previousBid;
+    
+    uint player1Balance;
+    uint player2Balance;
+         
+    address player1;
+    address player2;
+    
+    event playerJoined(address player, string message);
+    event Turn(address player, uint8 cell);
+    event Win(address winner, uint gain);
+    event Standoff();
+    event Error(string message);
+
+
+function makeTurn(uint8 cell) validBid public payable returns(string) {
```

Comment:
am I understand it right that player have to pay every move?
what will happen if a player suddenly make such a big bet that no one wants to join? Does player loose his money?
what will happen if one of the players suddenly die?